src/System.Linq.Queryable/src/System/Linq/Queryable.cs

```diff
@@ -39,10 +39,10 @@ public static IQueryable<TSource> Where<TSource>(this IQueryable<TSource> source
             return source.Provider.CreateQuery<TSource>(
                 Expression.Call(
                     null,
-                    GetMethodInfoOf(() => Queryable.Where(
+                    GetMethodInfoOf(() => Where(
```

Comment:
So it was in fact done as above before? A shame if it has issues, since I'd just come to the conclusion that it was the way to go :(